### ISSUE TITLE
Failure in setTsConfig while invoking `tsc --version`

### ISSUE THREAD

**Issue by foxbunny** [mhartington/nvim-typescript#118]
**Warning:  I will close the issue without the minimal vimrc and the reproduce ways.**

# Problems summary

When I open a `.ts` file on Windows, `nvim-typescript` plugin fails with the following: `subprocess.CalledProcessError: Command '['tsc --version']' returned non-zero exit status 1.

When I invoke it manually, like so: 

```
:py3 import subprocess; subprocess.check_output(['tsc --version'], shell=True)
```

I get the same traceback

If I use this instead:

```
:py3 import subprocess; subprocess.check_output(['tsc', '--version'], shell=True)
``` 

it works fine. Note that I've separated the command and argument.

## Expected

Expected not to get an error.

## Environment Information

 * terminal: Command Prompt (Win 10, nvim started with nvim-qt)
 * vim/nvim version: nvim v0.2.3-712-g422a788b8

## Provide a minimal vim rc with less than 50 lines (Required!)

I think the cause of the error is clear enough, so I won't be providing one here. If you still need one, let me know.

## The reproduce ways from neovim starting

 1. Open any .ts file

## Example project repo?

Will do if needed.

**Comment by mhartington** [comment]
Ahh, good catch. I will fix.

**Comment by mhartington** [comment]
This should be fixed in the `singleton` branch. Could you test it out and verify for me?

**Comment by mhartington** [comment]
Should be fixed and in master!

**Comment by foxbunny** [comment]
Hey, sorry, missed the notification. I'll check it out right away.
